bin/runtime/src/lib.rs

```diff
@@ -707,6 +711,41 @@ impl pallet_utility::Config for Runtime {
     type PalletsOrigin = OriginCaller;
 }
 
+parameter_types! {
+    // The following weights are pulled out of thin air. A separate task to adjust them is already in JIRA.
```

Comment:
maybe `///` instead of `//`?

Reply:
What's the difference? ^^'

Reply:
in the built docs: either some comment or none :P

Reply:
This comment is kinda internal and meant to be removed soon. Let's leave `//`